parl/utils/globvars.py

```diff
@@ -0,0 +1,44 @@
+#   Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from parl.utils import logger
+
+__all__ = ['global_config', 'GlobalConfig']
+
+import types
+CONFIG = types.SimpleNamespace
+
+class GlobalConfig(CONFIG):
+    """
+    A namespace to store global variables.
+    Example:
+    .. code-block:: none
+        from parl.utils.globvars import global_config as config
```

Comment:
It can be replaced by `from parl.utils import ...` after setting in parl/utils/__init__.py

Reply:
done.